lua/pac3/editor/client/panels/editor.lua

```diff
@@ -345,6 +349,10 @@ function pace.KillFocus(show_editor)
 			self:AlphaTo(0, 0.1, 0)
 			self.exit_button:AlphaTo(0, 0.1, 0)
 			self.zoomframe:AlphaTo(0, 0.1, 0)
+			
```

Comment:
This should probably have a 0.1 sec delay right? Since the AlphaTo animation would be pointless otherwise.

Reply:
Right, should be better now.

Reply:
Should probably make it a timer.create and do a timer.remove when the other case happens so turning it off and on again quickly won't hide it